src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -44,6 +44,20 @@ public NodeSearchElementViewModel(NodeSearchElement element, SearchViewModel svm
             ClickedCommand = new DelegateCommand(OnClicked);            
         }
 
+        /// <summary>
+        /// Creates a copy of NodeSearchElementViewModel.
+        /// </summary>
+        public NodeSearchElementViewModel(NodeSearchElementViewModel copyElement)
+        {
+            if (copyElement == null)
+                return;
```

Comment:
Shouldn't we be throwing an `ArgumentNullException` when `copyElement == null`? The behavior of this class is undefined if the caller did not supply a valid `copyElement`. Do you see a scenario when that can happen?